amazonorders/orders.py

```diff
@@ -13,8 +13,9 @@
 
 logger = logging.getLogger(__name__)
 
-ORDER_HISTORY_CARD_SELECTOR = "div[class*='order-card']:has(script)"
+ORDER_HISTORY_CARD_SELECTOR = "div[class*='js-order-card']"
```

Comment:
When I use `js-order-card` instead of `order-card` or `div.order`, I get two of every order instead of one (with `js-order-card` I also get a lot of warnings about recipient not being parsable, but `div.order` also works for me and doesn't cause any regressions.

Reply:
It doesn't work at all for me unless I have that, and I only get 1 copy of each.  I do get parsing warning as well. 

Maybe My account is different enough or like I said else where I may be in an A/B testing campaign.